packages/eslint-config-react-app/jest.js

```diff
@@ -12,21 +12,62 @@
 // and we try not to use "ERROR" level at all.
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['jest', 'jest-dom', 'testing-library'],
   overrides: [
     {
       files: ['**/__tests__/**/*', '**/*.{spec,test}.*'],
       env: {
         'jest/globals': true,
       },
       // A subset of the recommended rules:
-      // https://github.com/jest-community/eslint-plugin-jest#rules
       rules: {
-        'jest/expect-expect': 'warn',
-        'jest/no-identical-title': 'warn',
-        'jest/valid-describe': 'warn',
-        'jest/valid-expect': 'warn',
-        'jest/valid-expect-in-promise': 'warn',
+        // https://github.com/jest-community/eslint-plugin-jest
+        'jest/expect-expect': 'off',
+        'jest/no-commented-out-tests': 'warn',
+        'jest/no-conditional-expect': 'error',
+        'jest/no-deprecated-functions': 'warn',
+        'jest/no-disabled-tests': 'warn',
+        'jest/no-done-callback': 'error',
+        'jest/no-export': 'error',
+        'jest/no-focused-tests': 'warn',
+        'jest/no-identical-title': 'error',
+        'jest/no-interpolation-in-snapshots': 'error',
+        'jest/no-jasmine-globals': 'error',
+        'jest/no-jest-import': 'error',
+        'jest/no-mocks-import': 'error',
+        'jest/no-standalone-expect': 'off',
+        'jest/no-test-callback': 'off',
+        'jest/no-test-prefixes': 'warn',
+        'jest/valid-describe': 'error',
+        'jest/valid-expect': 'error',
+        'jest/valid-expect-in-promise': 'error',
+        'jest/valid-title': 'warn',
+
+        // https://github.com/testing-library/eslint-plugin-jest-dom
+        'jest-dom/prefer-checked': 'warn',
+        'jest-dom/prefer-empty': 'warn',
+        'jest-dom/prefer-enabled-disabled': 'warn',
+        'jest-dom/prefer-focus': 'warn',
+        'jest-dom/prefer-required': 'warn',
+        'jest-dom/prefer-to-have-attribute': 'warn',
+        'jest-dom/prefer-to-have-style': 'warn',
+        'jest-dom/prefer-to-have-text-content': 'warn',
+
+        // https://github.com/testing-library/eslint-plugin-testing-library
+        'testing-library/await-async-query': 'error',
+        'testing-library/await-async-utils': 'error',
+        'testing-library/no-await-sync-query': 'warn',
+        'testing-library/no-debug': 'warn',
+        'testing-library/no-dom-import': ['error', 'react'],
+        'testing-library/no-manual-cleanup': 'error',
+        'testing-library/no-render-in-setup': 'error',
+        'testing-library/no-wait-for-empty-callback': 'error',
+        'testing-library/no-wait-for-snapshot': 'error',
+        'testing-library/prefer-explicit-assert': 'warn',
+        'testing-library/prefer-find-by': 'warn',
+        'testing-library/prefer-presence-queries': 'warn',
+        'testing-library/prefer-screen-queries': 'warn',
+        'testing-library/prefer-wait-for': 'warn',
```

Comment:
I think we can also turn off:
- `testing-library/no-debug`
- `testing-library/no-manual-cleanup`
- `testing-library/no-render-in-setup`
- `testing-library/prefer-*`

I'm not sure about these, if they don't cause any errors - maybe we should turn them off?
- `testing-library/no-await-sync-query`
- `testing-library/no-dom-import`
- `testing-library/no-wait-for-empty-callback`

Reply:
For now I kept the `testing-library/no-await-sync-query`, `testing-library/no-dom-import` & `testing-library/no-wait-for-empty-callback` rules
Let me know if they need to be removed 🙂